libs/dh/reports/feature-measurements-reports/src/components/request-report/request-report-modal.component.ts

```diff
@@ -217,6 +227,7 @@ export class DhRequestReportModal extends WattTypedModal<MeasurementsReportReque
             end: period.end ? period.end : null,
           },
           gridAreaCodes: gridAreas,
+          preventLargeTextFiles: !allowLargeTextFiles,
```

Comment:
Question: Why is one called prevent and the other one allow. Seems odd?

Reply:
Because it's made exactly the same way as in settlement reports to keep the same pattern. A rename might follow for both reports to make the naming/param more clear.